index.js

```diff
@@ -288,3 +292,14 @@ function unregister(glob) {
   glob.Date = _Date;
 }
 exports.unregister = unregister;
+
+
+function setCurrentTime(timeToSet) {
+	currentTime = timeToSet;
+}
+exports.setCurrentTime = setCurrentTime;
```

Comment:
These functions aren't needed anymore?

Reply:
Yes you're right, removed them. Thanks for double checking!